docs/kubernetes/index.rst

```diff
@@ -1,140 +1,162 @@
+General Prerequisites
+`````````````````````
 
-.. sidebar:: :fonticon:`fa fa-exclamation-circle` Important:
+The |kctlr-long| documentation set assumes that you:
 
-   * The |kctlr| cannot manage objects in the ``/Common`` :term:`partition`.
-   * The BIG-IP partition you want to manage must exist before you launch the |kctlr|.
-   * The |kctlr| does not create or destroy BIG-IP partitions.
-   * You can use multiple |kctlr| instances to manage **separate** BIG-IP partitions.
-   * You can create one (1) BIG-IP virtual server per Service port.
-     *Create a separate* :ref:`virtual server F5 Resource ConfigMap <kctlr-create-vs>` *for each Service port you wish to expose.*
+- already have a Kubernetes `Cluster`_ running;
+- are familiar with the `Kubernetes dashboard`_ and `kubectl`_ ;
+- already have a BIG-IP device licensed and provisioned for your requirements; and
+- are familiar with BIG-IP LTM concepts and ``tmsh`` commands.
 
-- what `Service`_ it should manage, and
-- what objects it should create/update on the BIG-IP system for that Service.
+.. note::
 
-You can use F5 Resource ConfigMaps to deploy BIG-IP :ref:`virtual servers <kctlr-create-vs>` or :ref:`iApps <kctlr-deploy-iapps>`.
+   When using the |kctlr| in :ref:`cluster mode`, your BIG-IP license must include SDN services.
 
-The |kctlr| can:
+.. _k8s-installation:
 
-- :ref:`create a BIG-IP LTM virtual servers <kctlr-create-vs>` for a `Kubernetes Service`_
-- :ref:`use an IPAM system to assign IP addresses to virtual servers <kctlr-ipam>`
-- :ref:`create unattached pools <kctlr-pool-only>` (pools that aren't attached to virtual servers)
-- :ref:`deploy iApps <kctlr-deploy-iapps>`
-- act as a `Kubernetes Ingress controller`_ to :ref:`expose Kubernetes Services to external traffic <kctlr-ingress-config>`.
+Installation
+------------
 
+You can :ref:`launch the k8s-bigip-ctlr application <install-kctlr>` in Kubernetes using a Deployment.
 
-.. _k8s-prereqs:
+.. _k8s-upgrade:
 
-General Prerequisites
-`````````````````````
-
-The F5 Integration for Kubernetes documentation set assumes that you:
+Upgrade
+-------
 
-- already have a Kubernetes `cluster`_ running;
-- are familiar with the `Kubernetes dashboard`_ and `kubectl`_ ;
-- already have a BIG-IP :term:`device` licensed and provisioned for your requirements; and
-- are familiar with BIG-IP LTM concepts and ``tmsh`` commands.
+To upgrade an existing :code:`k8s-bigip-ctlr` instance to a newer version, take the steps below.
 
-.. note::
+#. :ref:`Edit the Controller Deployment <k8s-bigip-ctlr-deployment>`.
 
-   When using the |kctlr| in OpenShift, make sure your BIG-IP license includes SDN services.
+   - Update the :code:`image` property to use the desired version.
+   - Add/edit configuration parameters to use new features.
```

Comment:
I saw this in the cloud foundry docs this morning and it leaves me wanting. What are thoughts - perhaps link to the release notes here where we talk about new features? We could also link to the configuration section which kind of serves as a general overview of what the products do.

The thing I'm guarding against is someone reading this and seeing "new features" and reacting with "wtf new features". Suggestions welcome.

Reply:
I think I'd rather change the wording to something along the lines of "Add/edit the configuration parameters as desired". This can be improved/expanded upon in the future when we migrate all of the documentation into the project repos -- linking to places in the same project repo is easier than doing so across docs sourced in different projects.

Reply:
+1